.github/workflows/publish_image_chart.yaml

```diff
@@ -55,14 +55,14 @@ jobs:
 
       # Login to push container images.
       - name: Log in to the container registry
-        uses: docker/login-action@f054a8b539a109f9f41c372932f1ae047eff08c9
```

Comment:
could you add comment to show which version of login-action do we use for now 🙏

Reply:
Sure